### DISCUSSION TITLE
User not active

### DISCUSSION THREAD

**Discussion by rfilippus** [nilsteampassnet/TeamPass#4264] [General]
Goodnight.
I installed Teampass on my personal machine (mac) to check the features.
After creating the user, it displays the status "Taks in progress, User not active", and does not change. I've waited two days and no change.

In the TASK MANAGER option, I can see that there is a user addition record in the "In progress" tab.

What can I check?

Thank you for your attention.

**Comment by DobroFenix** [comment]
Run crontab 
manual or in dashbord (Information)

**Reply by rfilippus** [reply]
Good Morning

The crontab does not have any records, I checked with the commands below:
# crontab -l (list)
# cronta -e (edit)

I created a new user last night. Today, after 18 hours, I can see in "TASKS" that the PROGRESS bar is 86% complete.

What could be causing this slowness? I believe it is not normal.

Thanks

**Reply by rfilippus** [reply]
Task finish 10min ago, but the user continue with the same status.

To change de password, i receive the message: The account is currently being encrypted. Your request cannot continue. Please wait until this is complete.

I am thinking I need setup anything?
